merge_coda_files/merge_coda_files.py

```diff
@@ -32,12 +32,40 @@ def coda_id_to_strings(list_td, old_key, key_map, code_scheme):
     :type code_scheme: Scheme
     '''
     new_key = key_map[old_key]
+
     for td in list_td:
         if old_key in td:
-            td.append_data(
-                {new_key: code_scheme.get_code_with_id(td[old_key]["CodeID"]).string_value},
-                Metadata(user, Metadata.get_call_location(), time.time())
-            )
+            if type(td[old_key]) is list:
+                list_code_strings = list()
+                for code in td[old_key]:
+                    code_string = code_scheme.get_code_with_id(code["CodeID"]).string_value
+                    list_code_strings.append(code_string)
+                td.append_data(
+                    {new_key: list_code_strings},
+                    Metadata(user, Metadata.get_call_location(), time.time())
+                )
+            else:
+                td.append_data(
+                    {new_key: code_scheme.get_code_with_id(td[old_key]["CodeID"]).string_value},
+                    Metadata(user, Metadata.get_call_location(), time.time())
+                )
+
+def choose_coda_importer(is_multi_coded):
```

Comment:
Prefer something like `import_from_coda` because this function does more than just choose which importer to use - it also performs the actual import

Reply:
Changed to import_from_coda